javascript/javascript1/week3/Notepad.js

```diff
@@ -0,0 +1,65 @@
+let notes = [];
+
+function saveNote(content, id) {
+  // The function is to save notes like content (message)and id(like priority)
+
+  notes.push({ content, id });
+}
+function getNote(id) {
+  // the function is to find notes in the array of objects
+  if (id == null || isNaN(id)) {
+    return "getNote argument is null or not a number";
+  }
+  for (let i = 0; i < notes.length; i++) {
+    if (notes[i].id === id)
+      return "content is '" + notes[i].content + "' for the id " + notes[i].id;
+  }
+  return "content can not found for id " + id;
```

Comment:
the name of the function is `getNote` so it should return a note like this
```js
return notes[i];
```
and at line 17 you can log an error `console.error('content cannot found...)` or throw error like 
```js
throw new Error("content can not found for id " + id);
```
which is better